webapp/src/css/inbox.less

```diff
@@ -1506,7 +1506,7 @@ a.fa:hover {
   }
   .action-container .actions {
     min-height: 50px;
-    p {
+    .dropdown-toggle p {
```

Comment:
I think this has unintended consequences, for example, the labels on the buttons in the action bar are now shown, which is a regression on https://github.com/medic/cht-core/commit/9e52159fb5dcc2c46eca0dc70a580e07266cbb28 from 2.6.0.

Given the urgency of this issue I'm going to pick this up today and see if I can come up with an alternate solution.